package.json

```diff
@@ -19,6 +19,15 @@
     "body-parser": "latest",
     "bunyan": "latest",
     "bunyan-prettystream": "latest",
+    "clone": "latest",
```

Comment:
Is it really necessary to bring in `clone`?  Lodash has a `_.cloneDeep` function that does a lot of what `clone` does.